comid/tdx-profile/operator.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package tdx
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type Operator uint
+
+const (
+	EQ   = iota // EQual
+	GT          // greater than
+	GE          // greater than or equal to
+	LT          // less than
+	LE          // less than or equal to
+	NOP         // undefined
+	MEM         // member
+	NMEM        // no member
+	SUB         // sub-set
+	SUP         // super set
+	DIS         // dis-joint
+)
+
+var (
+	StringToNumericOperator = map[string]Operator{
+		"equal":            EQ,
+		"greater_than":     GT,
+		"greater_or_equal": GE,
+		"less_than":        LT,
+		"less_or equal":    LE,
+		"nop":              NOP,
+		"member":           MEM,
+		"non_member":       NMEM,
+		"sub_set":          SUB,
+		"super_set":        SUP,
+		"dis_joint":        DIS,
```

Comment:
Any reason for diverging from the spec, i.e.: `op.eq`, `op.gt`, etc. ?

Reply:
for the JSON now we are using string keys so perhaps does not make any difference.
For CBOR we use numbers.

I felt op. is repetitive for each operator and also aligning the json key names to logical name in the Operator connects well. Just a thought!

Reply:
> I felt op. is repetitive for each operator

Yes, I agree.  I’d go for short and matching `eq`, `gt`, `ge`, …, which are quite well-established shorthands in CS circles.